streaming_form_data/_parser.pyx

```diff
@@ -2,42 +2,59 @@ import cgi
 
 from streaming_form_data.targets import NullTarget
 
+ctypedef unsigned char Byte
+ctypedef size_t Index
```

Comment:
This alias was a bit confusing while reviewing. I would suggest that the type of the variables remains `size_t`.

Reply:
ok

Reply:
Can I do this in separate additional commit? So I don't have to resolve numerous conflicts

Reply:
Sure!